spec/requests/api/v1.2/portfolio_items_controller_spec.rb

```diff
@@ -32,6 +32,7 @@
         it 'return 200' do
           expect(response).to have_http_status(200)
           expect(json["id"]).to eq portfolio_item_id.to_s
+          expect(json["metadata"]["orderable"]).to eq(false)
```

Comment:
We can probably also make sure that `portfolio_item_orderable` does not receive the `process` message, which proves we are short circuiting.